### DISCUSSION TITLE
Magick++ exception for tiff library

### DISCUSSION THREAD

**Discussion by kritika-1720** [libvips/libvips#3607] [Q&A] [answered]
Hi,

We're using below version on our RHEL 8 VM:

Version: ImageMagick 7.1.0-53 Q16 x86_64 20646 https://imagemagick.org
Copyright: (C) 1999 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher
Delegates (built-in): jpeg lzma tiff zlib
Compiler: gcc (8.5)
Tiff version: 4.5.1

But facing the below in traces:

**Magick++ exception: Magick: ASCII value for tag "DocumentName" contains null byte in value; value incorrectly truncated during reading due to implementation limitations. `TIFFFetchNormalTag' @ warning/tiff.c/TIFFWarnings/951**

Can you please suggest what could be the issue, or any other resolution for this.

**Comment by jcupitt** [comment, marked as answer]
Hi @kritika-1720, I would guess a malformed TIFF image. But you'd need to share the image that triggered this warning.

libvips doesn't use magick++, it uses plain magickcore, so I'm not sure this is a libvips error.
